src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -192,6 +196,33 @@ public async Task UpdateStateAsync(IProgress<IItemLoaderState> progress, Cancell
             NuGetEventTrigger.Instance.TriggerEvent(NuGetEvent.PackageLoadEnd);
         }
 
+        private async Task<SearchResult<IPackageSearchMetadata>> CombineSearchAsync(
+            CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+
+            // get the browse/search results
+            SearchResult<IPackageSearchMetadata> browseResult = await _packageFeed.SearchAsync(_searchText, SearchFilter, cancellationToken);
+
+            // get the recommender results
+            SearchResult<IPackageSearchMetadata> recommenderResult = null;
+            if (_recommenderPackageFeed != null)
+            {
+                recommenderResult = await _recommenderPackageFeed.SearchAsync(_searchText, SearchFilter, cancellationToken);
+                _recommendedCount = recommenderResult.Count();
+            }
+
+            // update items list to include recommender results if needed
```

Comment:
What I think it actually does, and would be more clear
```suggestion
            // append recommender results if needed
```

Reply:
Changed to "if there are recommendations, add them to the top of the browse results list"